website/docs/r/spring_cloud_app.html.markdown

```diff
@@ -77,6 +77,8 @@ The following attributes are exported:
 
 * `id` - The ID of the Spring Cloud Application.
 
+* `fqdn` - The Fully Qualified DNS Name.
```

Comment:
Can we be  a little more verbose here? This is `*.azuremicroservices.io` on Azure public, which may not be obvious to users. I'm not sure if something else is used on sovereign / gov clouds?
Perhaps:
```suggestion
* `fqdn` - The Fully Qualified DNS Name of the Spring Application in the service.
```

Reply:
this is a `computed` field, which is used in the `azurerm_dns_cname_record` resource. I think there is no need in terraform side to concern the suffix, and actually the suffix is different for different clouds.